### PR TITLE
Extend login error message to mention OIDC config

### PR DESCRIPTION
**Pull request by rossf7**
@cokiengchiara Suggested adding this after the problem with bandicoot yesterday.

I think this is a pretty common case and so hopefully the error message improves things. WDYT?

### CODE REVIEW

**Review by axbarsan — APPROVED** [review]
Nice 👍

File: pkg/installation/info.go [hunk 1]
```diff
@@ -43,7 +43,7 @@ func getInstallationInfo(ctx context.Context, gqlClient graphql.Client) (install
 	var info installationInfo
 	err := gqlClient.ExecuteQuery(ctx, infoQuery, nil, &info)
 	if err != nil {
-		return installationInfo{}, microerror.Maskf(cannotGetInstallationInfoError, "make sure you're connected to the internet and that the Athena service is up and running\n%s", err.Error())
+		return installationInfo{}, microerror.Maskf(cannotGetInstallationInfoError, "make sure you're connected to the internet, OIDC is configured for the management cluster and that the Athena service is up and running\n%s", err.Error())
```
Comment (marians) [inline pkg/installation/info.go:46]:
```suggestion
		return installationInfo{}, microerror.Maskf(cannotGetInstallationInfoError, "make sure you're connected to the internet, OIDC authentication is configured for the Management API, and the Athena service is up and running\n%s", err.Error())
```

My suggestion is about adding `authentication` and replacing management cluster with `Management API`.

If that oxford comma doesn't make sense here, please fix as you think.

**Review by cokiengchiara — APPROVED** [review]
LGTM, thank you! I like Marian's suggestion.

File: pkg/installation/info.go (hunk 1 above)
Comment (rossf7) [reply pkg/installation/info.go:46]: Thanks Marian much better and good to add the oxford comma. 👍

**Pull request merged**